velox/common/base/BloomFilter.h

```diff
@@ -129,6 +135,58 @@ class BloomFilter {
     return mask == (bloom[index] & mask);
   }
 
+  inline static void testForBatch(
+      const uint64_t* FOLLY_NONNULL bloom,
+      int32_t bloomSize,
+      const uint64_t* hashCode,
+      bool* result,
+      int32_t size) {
+    VELOX_DCHECK_EQ(size, 64);
+#if XSIMD_WITH_AVX2
```

Comment:
Can this be written using xsimd API?

What are the gains? Would you include a benchmark that demonstrates the perf win?

CC: @Yuhta

Reply:
I'm not familiar with the xsimd APi, let me check.
As `might_contain` function are evlauted in remaining filters, we use remaining filter time to measure the perf gain, for details number please see #7363 . Let me also add some benchmark for this.
Thanks.

Reply:
It's very straightforward to rewrite this using xsimd, everything here is built in except `gather`, you can get it from `SimdUtil.h`